db/migrate/20230315130919_add_last_accessed_at_to_access_tokens.rb

```diff
@@ -0,0 +1,5 @@
+class AddLastAccessedAtToAccessTokens < ActiveRecord::Migration[7.0]
+  def change
+    add_column :access_tokens, :last_accessed_at, :datetime
+  end
+end
```

Comment:
I think its better keeping them separate because it shows the intent and keeps the commits separate. If you ever wonder what the schema looks like for any given PR that has migrations always check the db/schema.rb file